app/models/school.rb

```diff
@@ -22,6 +22,10 @@ class School < ApplicationRecord
 
   scope :open, -> { where(establishment_status_code: %w[1 3 4]) }
 
+  def display_name
+    name
+  end
```

Comment:
Are these for active admin?

Reply:
PrivateChildcareProviders sometimes have a blank name and so end up displaying "institution_identifier" as the radio text. To fix this I added display_name as a method on those for use in the no_js scenario. Since School and LocalAuthority also get rendered out in that view they needed to respond to the same interface.